ChiModules/LinearBoltzmannSolver/Tools/kspmonitor_npt.cc

```diff
@@ -81,28 +62,28 @@ KSPConvergenceTestNPT(KSP ksp, PetscInt n, PetscReal rnorm,
 
   chi_log.Log(LOG_0) << iter_info.str() << std::endl;
 
-  if (context->groupset->iterative_method == NPT_GMRES)
+  if (context->groupset.iterative_method == NPT_GMRES)
   {
     if (context->last_iteration == n)
     {
-      if (context->solver->options.write_restart_data)
+      if (context->solver.options.write_restart_data)
       {
         if ((chi_program_timer.GetTime()/60000.0) >
```

Comment:
Although not part of the pr, we should document the choice of 60000.0.

Reply:
More descriptive constant defined. This was intended to be an input parameter. Will create issue. For now, this is limited to an interval of 60s (60k milliseconds).